packages/types/src/api-reference/api-reference-configuration.ts

```diff
@@ -498,6 +498,7 @@ export type AnyApiReferenceConfiguration =
   | Partial<ApiReferenceConfiguration>
   | Partial<ApiReferenceConfigurationWithSources>
   | Partial<ApiReferenceConfigurationWithDefault>[]
+  | Partial<ApiReferenceConfigurationWithSources>[]
```

Comment:
```suggestion
  | Partial<ApiReferenceConfiguration>
  | Partial<ApiReferenceConfiguration>[]
```
should we just add sources and default as optional to the main schema?

Reply:
I don‘t know, I think I’d prefer to be more explicit there. I’d actually love the configuration to be more explicit in a few details (like requiring content OR url, currently everything is optional).

I’d tend to leverage TypeScript more, even if it’s just a detail.